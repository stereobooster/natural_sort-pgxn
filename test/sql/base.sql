\set ECHO none
BEGIN;
\i sql/natural_sort.sql
\set ECHO all

-- You should write your tests

SELECT natural_compare('test10', 'test1'), natural_compare('test1', 'test10'), natural_compare('test1', 'test1');

ROLLBACK;
