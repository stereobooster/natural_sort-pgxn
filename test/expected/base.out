\set ECHO none
-- You should write your tests
SELECT natural_compare('test10', 'test1'), natural_compare('test1', 'test10'), natural_compare('test1', 'test1');
 -1, 1, 0
--------------

(1 row)

ROLLBACK;
